nsqd/nsqd.go

```diff
@@ -444,7 +444,10 @@ func (n *NSQD) GetTopic(topicName string) *Topic {
 	// this makes sure that any message received is buffered to the right channels
 	lookupdHTTPAddrs := n.lookupdHTTPAddrs()
 	if len(lookupdHTTPAddrs) > 0 {
-		channelNames, _ := n.ci.GetLookupdTopicChannels(t.name, lookupdHTTPAddrs)
+		channelNames, err := n.ci.GetLookupdTopicChannels(t.name, lookupdHTTPAddrs)
+		if err != nil {
+			n.logf("ERROR: failed to get channels for pre-creation for topic %s - %s", t.name, err)
```

Comment:
Technically we failed to query all lookupd's so we don't know if the result is deficient in any way. We may have still gotten channels for pre-creation from available lookupd's. Rephrase this and/or make different error messages based on `channelNames`?